rss.py

```
@@ -0,0 +1,97 @@
+    paragraphs = soup.find_all("p")
+    description_paragraphs = []
+    for i, p in enumerate(paragraphs):
+        text = p.get_text(strip=True)
+
+        if not text:
+            continue
+        lower_text = text.lower()
+
+        if i == 0 and "getcomics" in lower_text:
+            continue
+        if i == len(paragraphs) - 1 and ("the post" in lower_text or "appeared first on" in lower_text):
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)